src/main/java/dev/dfonline/codeclient/command/impl/CommandDFGive.java

```diff
@@ -0,0 +1,84 @@
+import com.mojang.brigadier.builder.LiteralArgumentBuilder;
+import dev.dfonline.codeclient.ChatType;
+import dev.dfonline.codeclient.CodeClient;
+import dev.dfonline.codeclient.Utility;
+import dev.dfonline.codeclient.command.Command;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.command.CommandRegistryAccess;
+import net.minecraft.command.argument.ItemStackArgument;
+import net.minecraft.command.argument.ItemStackArgumentType;
+import net.minecraft.item.ItemStack;
+import net.minecraft.text.Text;
+
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.argument;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.literal;
+
+public class CommandDFGive extends Command {
+    @Override
+    public String name() {
+        return "dfgive";
+    }
+
+    @Override
+    public LiteralArgumentBuilder<FabricClientCommandSource> create(LiteralArgumentBuilder<FabricClientCommandSource> cmd, CommandRegistryAccess registryAccess) {
+        return cmd
+                .then(argument("item", ItemStackArgumentType.itemStack(registryAccess))
+                        .then(argument("count", IntegerArgumentType.integer(1))
+                                .executes(ctx -> {
+                                    giveItem(ctx.getArgument("item", ItemStackArgument.class)
+                                                    .createStack(1, false),
+                                            ctx.getArgument("count", Integer.class));
+                                    return 1;
+                                })
+                        )
+                        .executes(ctx -> {
+                            giveItem(ctx.getArgument("item", ItemStackArgument.class)
+                                    .createStack(1, false), 1);
+                            return 1;
+                        })
+                )
+                .then(literal("clipboard")
+                        .executes(ctx -> {
+                            if (CodeClient.MC.getNetworkHandler() == null) return -1;
+                            String clipboard;
+                            try {
+                                clipboard = CodeClient.MC.keyboard.getClipboard();
+                            } catch (Exception e) {
+                                Utility.sendMessage(Text.translatable("codeclient.command.dfgive.clipboard_unavailable"), ChatType.FAIL);
+                                return -1;
+                            }
+
+                            // Remove vanilla/dfgive command syntax, only leave the actual item and count arguments.
+                            // Every character will match in: '/dfgive @a '
+                            clipboard = clipboard.replaceAll("^/?(df)?(give )?(@[parens] )?", "");
+
+                            CodeClient.MC.getNetworkHandler().sendCommand("dfgive " + clipboard.trim());
```

Comment:
does it uuh account for having `/dfgive clipboard` and running `/dfgive clipboard`?

Reply:
I didn't think about that, even recode crashes lol